config/jobs/kubernetes/sig-release/cip/cloudbuilder.yaml

```diff
@@ -0,0 +1,20 @@
+promoter:
+  setup.sh: |
+    #!/bin/sh
```

Comment:
I have been advised in the past by Prow maintainers that scripts should actually live in the project and that Prow itself should just invoke the scripts by name.

Can you just move this script into the promoter repo itself?

Reply:
Yeah, if a script needs to exist, I would prefer to see this as a script inside of the repo, and then it could be invoked directly by the job as `scripts/golangci-lint` or something.  See https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-presubmits.yaml#L32 for example

I will admit this is neat, and did not realize this was a mode of passing a script "inline" that we could support.  The usual way I have seen this sort of thing done is with a `bash -c`, eg: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-testing/coverage.yaml#L27-L50